backend/dal/Services/Admin/Concrete/UserService.cs

```diff
@@ -53,32 +54,51 @@ public Paged<User> Get(int page, int quantity)
         public Paged<User> Get(UserFilter filter = null)
         {
             this.User.ThrowIfNotAuthorized(Permissions.AdminUsers);
+            var agencies = !this.User.IsInRole(Permissions.SystemAdmin.GetName())
```

Comment:
Use the cleaner extension method `HasPermission(Permissions.SystemAdmin)`.

Reply:
If you're going to return all agencies, don't filter on any of them.  It's a far better query.

Reply:
Generally it's best practice to introduce a statement as a positive condition, rather than a negative.  Change to `this.User.HasPermission(Permissions.SystemAdmin) ? positive : negative`

Reply:
You will still need to ensure the user can only view users that their permissions allow.  **System Administrators** can view everyone.  **Agency Administrators** can only view users within their agencies.

Reply:
Just to be clear, only sys-admin and agency-admin can get the user list?

Reply:
Correct, but only sys-admins can see everyone.